mobile_verifier/src/verifier.rs

```diff
@@ -109,15 +114,34 @@ impl VerifierDaemon {
         let speedtests =
             SpeedtestAverages::validated(&self.pool, scheduler.reward_period.end).await?;
 
-        let rewards = self.verifier.reward_epoch(heartbeats, speedtests).await?;
-
-        let mut owner_rewards: HashMap<Vec<u8>, u64> = HashMap::new();
-
-        for reward_share in rewards.into_radio_shares(&scheduler.reward_period)? {
-            *owner_rewards
-                .entry(reward_share.owner_key.clone())
-                .or_default() += reward_share.amount;
-
+        let poc_rewards = self.verifier.reward_epoch(heartbeats, speedtests).await?;
+        let mobile_price = self
+            .price_tracker
+            .price(&helium_proto::BlockchainTokenTypeV1::Mobile)
+            .await?;
+        // Mobile prices are supplied in 10^6, so we must convert them to Decimal
+        let mobile_price = Decimal::from(mobile_price) / dec!(1_000_000);
+        let transfer_rewards = TransferRewards::from_transfer_sessions(
+            mobile_price,
+            ingest::ingest_valid_data_transfers(
+                &self.verifier.file_store,
+                &scheduler.reward_period,
+            )
+            .await,
+            &scheduler.reward_period,
+        )
+        .await?;
+
+        // It's important to gauge the scale metric. If this value is < 1.0, we are in
+        // big trouble.
```

Comment:
if we're in _real_ trouble should we just bail the whole thing here? how much trouble are we talking? isn't that math dividing based on 40% of the total emissions available supposed to protect us from that trouble?